lib/y2_module_consoletest.pm

```diff
@@ -29,6 +29,26 @@ sub yast2_console_exec {
     }
 }
 
+sub ncurses_filesystem_probing {
+    my @needles = ('fs-probing-failed');
+    push @needles, pop();
+
+    assert_screen(\@needles, 300);
+    return unless (match_has_tag('fs-probing-failed'));
+
+    assert_screen 'fs-probing-details-btn';
+    send_key 'alt-d';
+
+    my $keymapping = {'probing-error' => 'ret', 'fs-probing-failed' => 'alt-i'};
+    foreach (qw(probing-error fs-probing-failed)) {
+        assert_screen $_;
+        send_key $keymapping->{$_};
+    }
+    # drop the probe needle
+    shift @needles;
```

Comment:
Sounds like having the argument(s) in a separate variable would be easier to read

Reply:
(Might also allow passing multiple tags, not sure how/whether perl's `push` handles that)

Reply:
Good point!

Reply:
> (Might also allow passing multiple tags, not sure how/whether perl's `push` handles that)

that would be the previous way. :D

Reply:
let's assume that we need only 1 exit needle per test case for now :D